package.json

```
@@ -22,10 +23,13 @@
     "color-string": "^1.9.0",
     "dompurify": "2.3.4",
     "dotenv": "^14.2.0",
+    "framer-motion": "^6.2.3",
     "gatsby": "^3.14.6",
+    "gatsby-link": "^4.6.0",
```

```suggestion
    "gatsby-link": "^3.14.0",
```

This fixes the issue locally for me. This project is still on Gatsby v3. I'm assuming gatsby-link `4.6.0` is for Gatsby v4

---

I dont understand why i had to even include gatsby-link the package says that it should be imported from gatsby. https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-link for this and a few other modules. 🤔